search.json

```diff
@@ -0,0 +1 @@
ttps://github.com/YetAnotherPackageInstaller/packages-darwin/blob/master/scripts/homebrew.sh"}},{"name":"macports.sh","path":"scripts/macports.sh","sha":"b5c546a4103c286cf8d246aad8a7f8776329aabb","size":343,"url":"https://api.github.com/repos/YetAnotherPackageInstaller/packages-darwin/contents/scripts/macports.sh?ref=master","html_url":"https://github.com/YetAnotherPackageInstaller/packages-darwin/blob/master/scripts/macports.sh","git_url":"https://api.github.com/repos/YetAnotherPackageInstaller/packages-darwin/git/blobs/b5c546a4103c286cf8d246aad8a7f8776329aabb","download_url":"https://raw.githubusercontent.com/YetAnotherPackageInstaller/packages-darwin/master/scripts/macports.sh","type":"file","_links":{"self":"https://api.github.com/repos/YetAnotherPackageInstaller/packages-darwin/contents/scripts/macports.sh?ref=master","git":"https://api.github.com/repos/YetAnotherPackageInstaller/packages-darwin/git/blobs/b5c546a4103c286cf8d246aad8a7f8776329aabb","html":"https://github.com/YetAnotherPackageInstaller/packages-darwin/blob/master/scripts/macports.sh"}},{"name":"test.sh","path":"scripts/test.sh","sha":"659ff074fcf2023fa18cacee3b7da5e1f2b89e2c","size":104,"url":"https://api.github.com/repos/YetAnotherPackageInstaller/packages-darwin/contents/scripts/test.sh?ref=master","html_url":"https://github.com/YetAnotherPackageInstaller/packages-darwin/blob/master/scripts/test.sh","git_url":"https://api.github.com/repos/YetAnotherPackageInstaller/packages-darwin/git/blobs/659ff074fcf2023fa18cacee3b7da5e1f2b89e2c","download_url":"https://raw.githubusercontent.com/YetAnotherPackageInstaller/packages-darwin/master/scripts/test.sh","type":"file","_links":{"self":"https://api.github.com/repos/YetAnotherPackageInstaller/packages-darwin/contents/scripts/test.sh?ref=master","git":"https://api.github.com/repos/YetAnotherPackageInstaller/packages-darwin/git/blobs/659ff074fcf2023fa18cacee3b7da5e1f2b89e2c","html":"https://github.com/YetAnotherPackageInstaller/packages-darwin/blob/master/scripts/test.sh"}},{"name":"wireshark.sh","path":"scripts/wireshark.sh","sha":"da57f2c65e7038afa752b4f9591563c06daf3c93","size":362,"url":"https://api.github.com/repos/YetAnotherPackageInstaller/packages-darwin/contents/scripts/wireshark.sh?ref=master","html_url":"https://github.com/YetAnotherPackageInstaller/packages-darwin/blob/master/scripts/wireshark.sh","git_url":"https://api.github.com/repos/YetAnotherPackageInstaller/packages-darwin/git/blobs/da57f2c65e7038afa752b4f9591563c06daf3c93","download_url":"https://raw.githubusercontent.com/YetAnotherPackageInstaller/packages-darwin/master/scripts/wireshark.sh","type":"file","_links":{"self":"https://api.github.com/repos/YetAnotherPackageInstaller/packages-darwin/contents/scripts/wireshark.sh?ref=master","git":"https://api.github.com/repos/YetAnotherPackageInstaller/packages-darwin/git/blobs/da57f2c65e7038afa752b4f9591563c06daf3c93","html":"https://github.com/YetAnotherPackageInstaller/packages-darwin/blob/master/scripts/wireshark.sh"}}]
```

Comment:
does this have to be here?

Reply:
search.json is a file that is received from the github api parsed for filenames in the scripts directory of each package directory. I will remove this file and add to .gitignore because it will be generated when the first search is ran.